docs/self-managed/setup/guides/air-gapped-installation.md

```diff
@@ -39,13 +39,16 @@ The following images must be available in your air-gapped environment:
 
 We currently have a script in the [camunda-helm-respository](https://github.com/camunda/camunda-platform-helm/blob/c6a6e0c327f2acb8746802fbe03b3774b8284de3/scripts/download-chart-docker-images.sh) that will assist in pulling and saving Docker images.
 
-Please note that all the required Docker images, available on DockerHub's Camunda organization, are also provided publicly via Camunda's Docker registry: `registry.camunda.cloud/camunda/<image>`
+Please note that all the required Docker images, available on DockerHub's Camunda and Bitnami organizations, are also provided publicly via Camunda's Docker registry: `registry.camunda.cloud/camunda/<image>` and `registry.camunda.cloud/bitnami/<image>`
```

Comment:
I've always missed a header before this paragraph, for the ability to make a deeplink to the segment on this page allowing to the Harbor as a mirror option, wdyt?

Something like:
```
## Accessing Camunda images from Camunda Docker Registry
```

Reply:
I like it, thank you!